hicommonwealth/commonwealth
# UI for Farcaster

## Link to Issue
Closes: #7661

## Description of Changes
- added feature flag for farcaster
- modified existing flow for launching contest so that farcaster contest can be selected and launched
- this is only UI, farcaster contests do not work yet

## Test Plan
- add FLAG_FARCASTER_CONTEST= true to your .env
- click `create contest`
- you should see screen when either common or farcaster contest type can be selected
- when common selected, nothing changes
- when farcaster selected, the form is a bit different
  - added description
  - added dropdown with days
  - removed recurring and topics section


https://github.com/user-attachments/assets/3f169cf2-56ef-46d3-9280-41dd06ce3654



## Deployment Plan

1. n/a

## Other Considerations

- n/a
